code/modules/mob/living/carbon/superior_animal/defense.dm

```diff
@@ -0,0 +1,286 @@
+/mob/living/carbon/superior_animal/proc/harvest(var/mob/user)
+	var/actual_meat_amount = max(1,(meat_amount/2))
+	if(meat_type && actual_meat_amount>0 && (stat == DEAD))
+		for(var/i=0;i<actual_meat_amount;i++)
+			var/obj/item/meat = new meat_type(get_turf(src))
+			meat.name = "[src.name] [meat.name]"
+		if(issmall(src))
+			user.visible_message(SPAN_DANGER("[user] chops up \the [src]!"))
+			new/obj/effect/decal/cleanable/blood/splatter(get_turf(src))
+			qdel(src)
+		else
+			user.visible_message(SPAN_DANGER("[user] butchers \the [src] messily!"))
+			gib()
+
+/mob/living/carbon/superior_animal/bullet_act(var/obj/item/projectile/P, var/def_zone)
+	. = ..()
+
+	updatehealth()
+
+/mob/living/carbon/superior_animal/attackby(obj/item/I, mob/living/user, var/params)
+	if (meat_type && (stat == DEAD) && (QUALITY_CUTTING in I.tool_qualities))
+		if (I.use_tool(user, src, WORKTIME_NORMAL, QUALITY_CUTTING, FAILCHANCE_NORMAL, required_stat = STAT_BIO))
+			harvest(user)
+	else
+		. = ..()
+		updatehealth()
+
+/mob/living/carbon/superior_animal/resolve_item_attack(obj/item/I, mob/living/user, var/hit_zone)
+	//mob.attackby -> item.attack -> mob.resolve_item_attack -> item.apply_hit_effect
+	return 1
+
+/mob/living/carbon/superior_animal/attack_hand(mob/living/carbon/M as mob)
+	..()
+	var/mob/living/carbon/human/H = M
+
+	switch(M.a_intent)
+		if (I_HELP)
+			help_shake_act(M)
+
+		if (I_GRAB)
+			if(M == src || anchored)
+				return 0
+			for(var/obj/item/weapon/grab/G in src.grabbed_by)
+				if(G.assailant == M)
+					M << SPAN_NOTICE("You already grabbed [src].")
+					return
+
+			var/obj/item/weapon/grab/G = new /obj/item/weapon/grab(M, src)
+			if(buckled)
+				M << SPAN_NOTICE("You cannot grab [src], \he is buckled in!")
+			if(!G) //the grab will delete itself in New if affecting is anchored
+				return
+
+			M.put_in_active_hand(G)
+			G.synch()
+			LAssailant = M
+
+			M.do_attack_animation(src)
+			playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+			visible_message(SPAN_WARNING("[M] has grabbed [src] passively!"))
+
+			return 1
+
+		if (I_DISARM)
+			playsound(loc, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
```

Comment:
Is it not possible for disarming to actually accomplish anything, like pushing the target away?

Reply:
I'm open to suggestions but pushing away will not accomplish anything due to how their current AI works, they would just immediately return to their previous tile. I can refine this in a future PR with some chance to make them bugger off for a bit so people can run away.